js/app.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const oReqVader = new XMLHttpRequest();
+oReqVader.addEventListener('load', reqListenerVaderName);
+oReqVader.open('GET', 'https://swapi.co/api/people/4/');
+oReqVader.send();
+
+function reqListenerVaderName() {
+  let nameOf4 = document.getElementById('person4Name');
+  nameOf4.innerHTML = JSON.parse(this.responseText).name;
+}
+
+const oReqVaderHomeWorld = new XMLHttpRequest();
+oReqVaderHomeWorld.addEventListener('load', reqListenerVaderWorld);
+oReqVaderHomeWorld.open('GET', 'https://swapi.co/api/planets/1/');
+oReqVaderHomeWorld.send();
```

Comment:
Getting the person's planet should be done from within the `reqListenerVaderName`